vulns/pip/PYSEC-0000-CVE-2021-3572.yaml

```diff
@@ -0,0 +1,22 @@
+id: PYSEC-0000-CVE-2021-3572
+details: A flaw was found in python-pip in the way it handled Unicode separators in
+  git references. A remote attacker could possibly use this issue to install a different
+  revision on a repository. The highest threat from this vulnerability is to data
+  integrity. This is fixed in python-pip version 21.1.
+affected:
+- package:
+    name: pip
+    ecosystem: PyPI
+    purl: pkg:pypi:pip
+  ranges:
+  - type: ECOSYSTEM
+    events:
+      introduced: "0"
+      fixed: 21.1
```

Comment:
This probably needs to be `"21.1"` for it to be interpreted as a string.

Reply:
Thanks!